core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -496,68 +646,106 @@ public String toString() {
   public static final PropertyKey MASTER_RETRY =
       new Builder(Name.MASTER_RETRY)
           .setDefaultValue(String.format("${%s}", Name.USER_RPC_RETRY_MAX_NUM_RETRY))
+          .setDescription("The number of retries that the client connects to master. (NOTE: "
+              + "this property is deprecated, use `alluxio.user.rpc.retry.max.num.retry` "
```

Comment:
instead of hard coding "alluxio.user.rpc.retry.max.num.retry" , use `Name.USER_RPC_RETRY_MAX_NUM_RETRY`